### PR TITLE
Bump buildah images

### PR DESCRIPTION
**Pull request by Divyansh42**
# Changes

Buildah image in few tasks was not
pointing to latest image release tag, so updated the image to point to
latest release.

Signed-off-by: Divyansh42 <diagrawa@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [ ] Complies with [Catalog Orgainization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [ ] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [ ] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [ ] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [ ] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [ ] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

---

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]:  https://github.com/tektoncd/catalog/issues/413

### CODE REVIEW

File: task/aws-ecr-login/0.1/samples/test-task.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ spec:
       type: string
   steps:
     - name: pull-and-push
-      image: quay.io/buildah/stable:v1.14.8
+      image: quay.io/buildah/stable
```
Comment (chmouel) [inline task/aws-ecr-login/0.1/samples/test-task.yaml:19]: we do want to have the version in there (And everywhere), isnt that the best practices for images or something with the tag ? cc @vdemeester

Comment (Divyansh42) [reply task/aws-ecr-login/0.1/samples/test-task.yaml:19]: Indeed then probably we need to fix [here](https://github.com/tektoncd/catalog/blob/master/task/s2i/0.1/s2i.yaml#L64) as well. :slightly_smiling_face:

**Review by vinamra28 — APPROVED** [review]
/lgtm

**Comment by piyush-garg** [comment]
@Divyansh42 If we want to change the buildah image version, can we do for all places where buildah image is used like buildah task, etc.

**Comment by Divyansh42** [comment]
> @Divyansh42 If we want to change the buildah image version, can we do for all places where buildah image is used like buildah task, etc.

@piyush-garg Yeah sure, will do that.

**Comment by chmouel** [comment]
> @Divyansh42 If we want to change the buildah image version, can we do for all places where buildah image is used like buildah task, etc.

I wonder if that's something that catlin can detect, detect if we have different version  of image.. I have added this issue https://github.com/tektoncd/plumbing/issues/570

**Review by vdemeester — APPROVED** [review]
/meow

**Comment by Divyansh42** [comment]
I have something really wrong while rebase and merge. 
Closing this PR and Reponed another PR https://github.com/tektoncd/catalog/pull/510

**Pull request closed without merging**